src/glewlwyd.c

```diff
@@ -274,6 +274,10 @@ int main (int argc, char ** argv) {
 void exit_server(struct config_elements ** config, int exit_value) {
 
   if (config != NULL && *config != NULL) {
+    /* stop framework */
+    ulfius_stop_framework((*config)->instance);
+    ulfius_clean_instance((*config)->instance);
+    o_free((*config)->instance);
```

Comment:
This change makes sense, I should stop the webserver and the database connection before cleaning the config structure.